### ISSUE TITLE
mapping options should be restricted to the targeted/focus clade?

### ISSUE THREAD

**Issue by rgazis** [OpenTreeOfLife/opentree#534]
[not a big deal]

but when mapping, a terminal which did not mapped automatically, suggest me two options 

http://tree.opentreeoflife.org/curator/study/edit/ot_234/?tab=otu-mapping
Graminella sp. NOR-54-1

![screen shot 2015-01-08 at 11 57 12 am](https://cloud.githubusercontent.com/assets/6391539/5666439/859d04ce-972d-11e4-9620-1bec6e7cd932.png)

One is the correct option (within Fungi) and the other one is in an insect family. I was wondering if it would be a good idea to include the Kingdom in the options to avoid errors when mapping.

Romina

**Comment by jimallman** [comment]
In the Mapping Options box, we allow the curator to choose a narrower "search context" (is this what you mean?)

![screen shot 2015-01-08 at 12 12 54 pm](https://cloud.githubusercontent.com/assets/446375/5666738/cb861172-972f-11e4-8a7c-2d8d9bc0ed10.png)

**Comment by rgazis** [comment]
yes! but have that selected and still get both options…maybe it is not working? 
R

**Comment by jimallman** [comment]
Hm, it worked in my quick test. Perhaps you need to "clear" the mapping (by clicking the 'X' button alongside those two options)?

**Comment by rgazis** [comment]
It works, I needed to clear the field before. Although "Fungi" was already selected before I started mapping...

**Comment by jimallman** [comment]
> Although "Fungi" was already selected before I started mapping...

That's odd, I can't reproduce this. I just tried the same operation. As you say, "Fungi" is already selected when I open the study. But for me, editing a label to "Graminella" and mapping it gives just a single result.

![screen shot 2015-01-08 at 12 52 49 pm](https://cloud.githubusercontent.com/assets/446375/5667374/543dc7bc-9735-11e4-890e-a88c8cf7739e.png)
